### DISCUSSION TITLE
Do you need the watch, does it do anything different?

### DISCUSSION THREAD

**Discussion by YorkshiremanJim** [Taxel/PlexTraktSync#1528] [Q&A] [answered]
If you have the sync set up to run every 3 hours for example, it will look at what's been played in plex, anything new etc and update Trakt with the play history.. 
Does watch just do the same thing but quicker?   So do you need it if you're happy that there's a 3 hour delay possibly in the update, or doesn't it have extra functionality? 

Thank you :-)

**Comment by simonc56** [comment, marked as answer]
`watch` also set your trakt watching status (if you're currently watching something) to : `watching Indiana Jones and the Last Crusade`

**Reply by glensc** [reply]
also, watch is able to handle rewatches.

**Reply by glensc** [reply]
and sometimes watch is missing things for various reasons. too slow. connection errors. so resync time to time can fix those issues.

also watch can't do ratings, liked lists. watch is only useful for marking watched status. adding to collection also sometimes works. main idea behind watch command is scrobbling.
